### PR TITLE
Condition Category Classification Code

### PR DESCRIPTION
**Pull request by sydneanschutz**
Description
----
Return Condition Category Classification Code

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

Screen Shot 2021-07-15 at 12 41 13 PM
Screen Shot 2021-07-15 at 12 41 38 PM
Screen Shot 2021-07-15 at 12 41 50 PM
Screen Shot 2021-07-15 at 12 42 05 PM

### CODE REVIEW

File: lib/resources/r4/condition.yaml [hunk 1]
```diff
@@ -167,14 +168,21 @@ fields:
     }
   binding:
     description: A category assigned to the condition.
```
Comment (RayGunY) [inline lib/resources/r4/condition.yaml:170]: On the jira it says to update this to "A classification assigned......" is that what we want or did we change our minds to keep it as category?

Comment (sydneanschutz) [reply lib/resources/r4/condition.yaml:170]: this description is for the field category, like in general. That description is for specifically the classification code. not sure where i would put it

File: lib/resources/example_json/r4_examples_condition.rb [hunk 2]
```diff
@@ -237,6 +274,17 @@ module Resources
             }
           ],
           'text': 'Health Concern'
+        },
+        {
```
Comment (MadhurRaj) [inline lib/resources/example_json/r4_examples_condition.rb:278]: health concerns won't the classification.

File: lib/resources/example_json/r4_examples_condition.rb [hunk 3]
```diff
@@ -384,6 +443,17 @@ module Resources
               }
             ],
             'text': 'Health Concern'
+          },
+          {
```
Comment (MadhurRaj) [inline lib/resources/example_json/r4_examples_condition.rb:447]: same here.

**Review by MadhurRaj — CHANGES_REQUESTED** [review]

File: lib/resources/example_json/r4_examples_condition.rb (hunk 2 above)
Comment (sydneanschutz) [reply lib/resources/example_json/r4_examples_condition.rb]: https://github.com/cerner/fhir.cerner.com/pull/701/commits/b8983bad284d00a1972186571002ee91b6559052#diff-20bf74979e847ab2cdb40b65f16e39069096d8a3a39fefe73adeae819dc6bbdfL241

File: lib/resources/example_json/r4_examples_condition.rb (hunk 3 above)
Comment (sydneanschutz) [reply lib/resources/example_json/r4_examples_condition.rb]: https://github.com/cerner/fhir.cerner.com/pull/701/commits/b8983bad284d00a1972186571002ee91b6559052#diff-20bf74979e847ab2cdb40b65f16e39069096d8a3a39fefe73adeae819dc6bbdfL411

File: lib/resources/r4/condition.yaml (hunk 1 above)
Comment (RayGunY) [reply lib/resources/r4/condition.yaml:170]: Ok fair enough then, not too worried about it

**Review by RayGunY — APPROVED** [review]

**Review by MadhurRaj — APPROVED** [review]

**Comment by sydneanschutz** [comment]
Verified on live
Screen Shot 2021-07-26 at 11 50 29 AM

**Pull request merged**